.github/actions/deploy-template-operator/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
       shell: bash
       run: |
         make build-manifests
-        kyma alpha create module --module-config-file ./module-config.yaml --path . --registry localhost:5111 --insecure
+        module create --module-config-file ./module-config.yaml --registry http://localhost:5111 --insecure
```

Comment:
I suggest that we keep the Kyma CLI for this action since we agreed that the module teams will still be using the CLI until they have their modulereleasemeta in place. Also, this action will be removed after all modules are already using the modulereleasemeta.

Also, one more thing, using the modulectl generates the new format of the ModuleTemplate (and later on a different name <modulename-moduleversion>), and this will not be similar to the real modules' ModuleTemplates on landscapes

Reply:
As agreed, we will keep using the modulectl here to remove the Kyma-cli actions, and the ModuleTemplates generated by kyma CLI (the ones in tests/moduletemplates) are already used in some tests, so we still cover cases with the old OCM layers format